### ISSUE TITLE
ListAccounts cannot deserialize

### ISSUE THREAD

**Issue by danhoffman699** [wyyerd/stripe-rs#114]
I'm using version 0.11.0, and the following
```
stripe::Account::list(client, stripe::ListAccounts::default()).unwrap().get_all(client).unwrap()
```
gives me
```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Deserialize(Error("invalid type: null, expected i64", line: 214, column: 21))', src/libcore/result.rs:1165:5
```

Any ideas? Thanks

**Comment by danhoffman699** [comment]
Closing this beca

**Comment by kestred** [comment]
I've added a test for this in #100; so should be resolved after that MR has a passing test suite and is merged.

**Comment by kestred** [comment]
Fixed in https://github.com/wyyerd/stripe-rs/pull/100/commits/d71e1911cbfc6500824538be3b5bbd7e422983e1 — https://github.com/wyyerd/stripe-rs/pull/100/commits/1ec8012cc97103965a782907c49459563a629c87.

Now on master and will be published as part of `0.12.0`.
